src/signals/incident-management/containers/FilterTagList/index.js

```diff
@@ -50,28 +53,45 @@ const renderTag = (key, mainCategories, list) => {
     >
       {display}
     </StyledTag>
-  );
-};
+  ), []);
 
-export const FilterTagListComponent = props => {
-  const {
-    tags,
-    dataLists,
-    categories: { main, sub },
-  } = props;
+  const renderGroup = useCallback((tag, list, tagKey) => {
```

Comment:
Might also be better to take both `renderGroup` and `renderTag` out of the component context.